src/SayMore/Transcription/UI/SegmentingAndRecording/SegmenterDlgBase.cs

```diff
@@ -286,6 +286,11 @@ protected Rectangle GetPlayOrigButtonRectangleForSegment(Rectangle rc)
 			if (rc.IsEmpty || _playButtonSize.Width + 6 > rc.Width)
 				return Rectangle.Empty;
 
+			if (rc.Width < 45)
+				return new Rectangle(rc.X + 3,
```

Comment:
Similarly, the 3 and 6 here should be constants to ensure everything stays aligned. (Or make the aforementioned helper method smart enough to take a collection of "other" buttons that need to fit on the same row. In this case, there would be no others. Not sure though - that might over-complicate the logic and make it less readable.)